mcodingbot/plugins/info.py

```diff
@@ -9,22 +9,20 @@
 
 @plugin.include
 @crescent.command(name="ping", description="Pong!")
-class PingCommand:
-    async def callback(self, ctx: Context) -> None:
-        await ctx.respond(f"Pong! {round(ctx.app.heartbeat_latency*1000)} ms.")
+async def ping(ctx: Context) -> None:
+    await ctx.respond(f"Pong! {round(ctx.app.heartbeat_latency*1000)} ms.")
 
 
 @plugin.include
 @crescent.command(name="links", description="Useful links")
-class LinksCommand:
-    async def callback(self, ctx: Context) -> None:
-        embed = hikari.Embed(
-            title="Links",
-            color=CONFIG.theme,
-            description=(
-                f"**[mCoding Youtube]({CONFIG.mcoding_youtube})**\n"
-                f"**[mCoding repo]({CONFIG.mcoding_repo})**\n"
-                f"**[mCodingBot repo]({CONFIG.mcodingbot_repo})**"
-            ),
-        )
-        await ctx.respond(embed=embed)
+async def links(ctx: Context) -> None:
+    embed = hikari.Embed(
+        title="Links",
+        color=CONFIG.theme,
+        description=(
+            f"**[mCoding Youtube]({CONFIG.mcoding_youtube})**\n"
```

Comment:
Also, I suggest changing `Youtube` to `YouTube`
```suggestion
            f"**[mCoding YouTube]({CONFIG.mcoding_youtube})**\n"
```

Reply:
or even `YouTube channel`

Reply:
this is outside the scope of this pr